fees/buffer/index.ts

```diff
@@ -18,22 +18,23 @@ export function _getDayId(timestamp: number): string {
 const graphs = (graphUrls: ChainEndpoints) => {
   return (chain: Chain) => {
     return async (timestamp: number) => {
-      const dateId = _getDayId(timestamp);
+      const dayTimestamp = timestamp - 86400;
+      const dateId = _getDayId(dayTimestamp);
```

Comment:
why use dateId (08/07) instead (09/07) ?

Reply:
Because as per our stats, we log data for the day 4pm onwards. So currently defillama shows our data of a certain day from 4pm-12am of that day. That is, only 8 hours.

Reply:
If we take the former day then that way we will have complete data

Reply:
okay